Update 2024-08-09-anaconda.md (#432)
sunpy/sunpy.org: posts/2024/2024-08-09-anaconda.md

# Anaconda packages are not "free"

> We wanted to inform the wider community about [Anaconda Inc](https://www.anaconda.com/) and if you are legally allowed to use their products for free.

Author: ```suggestion
We wanted to inform the wider community about [Anaconda Inc.](https://www.anaconda.com/) and if you are legally allowed to use their products for free.
```

> What is not commonly known is that Anaconda Inc. has implented specific clauses in its software licenses which determine if one is able to use it for free, based on the organization you work for.

Author: ```suggestion
What is not commonly known is that Anaconda Inc. has implemented specific clauses in its software licenses which determine if one is able to use Anaconda or Anaconda Navigator for free, based on the organization you work for.
```

[Anaconda themselves summarize this in a really detailed blog post](https://www.anaconda.com/blog/is-conda-free), which we use here to form the basis of this post.

> This also applies to the Anaconda Navigator which is a desktop graphical user interface (GUI) that many people use.

Author: ```suggestion
This also applies to the [Anaconda Navigator](https://docs.anaconda.com/navigator/) which is a desktop graphical user interface (GUI) that many people use.
```

> In addition, we're looking into a flexible pricing structure for non-profit organizations with over 200 full-time employees

> Hopefully, by the end of the year with these changes in place, users will have a clear understanding if they are allowed to use Anaconda Inc products for free.

Author: ```suggestion
Hopefully, by the end of the year with these changes in place, users will have a clear understanding if they are allowed to use Anaconda Inc. products for free.
```

## What is the alternative?

The answer from the SunPy Project is the following:

- Use the "conda-forge" channel provided by the conda-forge project.
  conda-forge is a community-led project which creates recipes, hosts infrastructure and distributions for use with `conda`.
  They provide a distribution (similar to `miniconda`) called [miniforge](https://github.com/conda-forge/miniforge) is configured to use the "conda-forge" channel by default.
  It is also the only way to [install any SunPy Project library](https://docs.sunpy.org/en/stable/tutorial/installation.html#installing-miniforge) via `conda`.
- Never setup or use the "defaults" channel if you install `miniforge`.

A blog post written by Tim de Jager & Ruben Arts [summarizes how conda-forge is free and avoids any vendor lock in.](https://prefix.dev/blog/towards_a_vendor_lock_in_free_conda_experience)

> ## I have Anaconda or Navigator already - What should I do?

Author: ```suggestion
## I have Anaconda or Anaconda Navigator already - What should I do?
```
